### PR TITLE
Registration Api

### PR DESCRIPTION
**Pull request by shahriaralamsakib**

### CODE REVIEW

File: home/__pycache__/__init__.cpython-310.pyc
Comment (ofsazib) [inline home/__pycache__/__init__.cpython-310.pyc:1]: @shahriaralamsakib Remove all `.pyc` files. The directory `__pycache__` should be added in `/gitignore`

File: home/admin.py
Comment (ofsazib) [inline home/admin.py:1]: @shahriaralamsakib  The app we create for users/authentication/permission is commonly named as core

File: home/models.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
 from django.db import models
 
 # Create your models here.
+
+class UserInfo(models.Model):
```
Comment (ofsazib) [inline home/models.py]: @shahriaralamsakib  Keep the model name `User` and it should be inherited from Django's abstructbase user model. Get help from internet what is abstrucrbaseuser model

File: home/urls.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from .views import UserRegistrationView, UserInfoView, AllUsersView
+
+urlpatterns = [
+    path('register/', UserRegistrationView.as_view(), name='user-registration'),
+    path('user/<int:pk>/', UserInfoView.as_view(), name='user-info'),
+     path('users/', AllUsersView.as_view(), name='all-users'),
```
Comment (ofsazib) [inline home/urls.py]: @shahriaralamsakib Fix indentation.

File: home/models.py (hunk 1 above)
Comment (ofsazib) [reply home/models.py]: There should be 2 types of user,
1. Cusromer
2. Admin
3. Super User

You can handle the types using a textchoice field

File: rokomari_c/urls.py [hunk 3]
```diff
@@ -19,5 +19,6 @@
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api/', include('home.urls')),
```
Comment (ofsazib) [inline rokomari_c/urls.py]: @shahriaralamsakib  We should have separate url files for each app. Those urls file will be linked in the main urls file

**Review by ofsazib — CHANGES_REQUESTED** [review]

File: home/admin.py
Comment (shahriaralamsakib) [reply home/admin.py:1]: done

File: home/__pycache__/__init__.cpython-310.pyc
Comment (shahriaralamsakib) [reply home/__pycache__/__init__.cpython-310.pyc:1]: done

File: home/urls.py (hunk 2 above)
Comment (shahriaralamsakib) [reply home/urls.py]: done

**Pull request merged**